src/translations/es.json

```diff
@@ -0,0 +1,62 @@
+{
+  "camera": {
+    "all_cameras": "Todas las Vámaras",
+    "cameras": "Cámaras"
+  },
+  "climate": {
+    "all_climates": "Todos los Aires Acondicionados",
+    "climates": "Aires Acondicionados"
+  },
+  "cover": {
+    "all_covers": "Todas las Cortinas",
+    "covers": "Cortinas"
```

Comment:
It's not just about curtains, it's about rollershutters, blinds, garage doors, etc.
See https://www.home-assistant.io/integrations/cover/